app/code/community/Reverb/ReverbSync/Helper/Orders/Creation/Address.php

```diff
@@ -22,8 +26,10 @@ public function addOrderAddressAsShippingAndBillingToQuote(stdClass $reverbOrder
         $shippingAddressObject = $reverbOrderObject->shipping_address;
         if (!is_object($shippingAddressObject))
         {
-            $error_message = $this->__(self::ERROR_NO_ADDRESS);
-            throw new Exception($error_message);
+            // In this event, we still want to create the Magento order. We will populate the address fields with a
+            //      fake placeholder value
+            $this->_addNoAddressToOrderObject($reverbOrderObject);
+            $shippingAddressObject = $reverbOrderObject->shipping_address;
```

Comment:
Sounds good. I'll try to get to that today but I might not be able to until tomorrow/this weekend. Can you open an issue for it please?